explorer/lib/explorer/models/batches.ex

```diff
@@ -14,15 +14,16 @@ defmodule Batches do
     field :response_transaction_hash, :string
     field :response_timestamp, :utc_datetime
     field :data_pointer, :string
+    field :proof_hashes, {:array, :string}
```

Comment:
Maybe `proof_hashes` should be stored using an 1-N relationship table.
It brings several improvements
- Hashes are indexables, so you can search for a proof and get the associated batch
- If you need to add more info to the proofs, a second table would be easier to use